test/rosdep_repo_check/config.yaml

```diff
@@ -44,6 +51,8 @@ supported_versions:
   fedora:
     - '33'
     - '34'
+  opensuse:
+    - '15.2'
```

Comment:
Note that this needs to be kept up-to-date by someone when the target version changes.